bindings/rust/sys/src/bindings.rs

```diff
@@ -0,0 +1,4624 @@
+/* automatically generated by rust-bindgen 0.71.1 */
```

Comment:
As far as I know how `bindgen` works, we shall not include this in repo directly? Maybe just include the artifacts in the `OUT_DIR`?

Reply:
Hm, I'm not against not checking it in, but not everyone that's using Rust might have Cmake and other dependencies installed (otherwise, we could make it an explicit requirement in the readme)

Reply:
I do think we already require `cmake` and `cc` to build rust bindings. We can make it explicit in readme or give a hint message when we fail to find any in `build.rs`.

Reply:
done